salesagility/SuiteCRM
# Fix/4947 workflow datetime field

Makes datetime fields display as datetimecombo fields with hour and minute dropdowns.

## Description



Fixes #4947 

Previously in both AOW Workflow and AOR Reports condition fields of type datetime and datetimecombo were displayed as a simple input which allowed only to choose the date, but not the time.

![screenshot from 2018-01-17 15-48-39](https://user-images.githubusercontent.com/34056696/35051766-e6cc5010-fb9d-11e7-9f6a-2257b191167f.png)

With this change the field has been expanded into datetimecombo field which allows more precise filtering.

![screenshot from 2018-01-17 11-29-52](https://user-images.githubusercontent.com/34056696/35051852-135678ae-fb9e-11e7-89fc-107db514988e.png)

![screenshot from 2018-01-17 15-39-05](https://user-images.githubusercontent.com/34056696/35051874-1d35990e-fb9e-11e7-88c4-b31cd05cc800.png)

![screenshot from 2018-01-17 15-39-18](https://user-images.githubusercontent.com/34056696/35051881-22b56c9c-fb9e-11e7-95d0-a75014e60065.png)

## How To Test This

1. Rebuild CSS
2. Go to either Reports or Workflow module.
3. Choose any related module
4. Set a condition based on a datetime field
5. See that the filters take into account the time

## Types of changes


### Final checklist
